articles/quickstart/spa/vuejs/02-calling-an-api.md

```diff
@@ -0,0 +1,234 @@
+  - vuejs
+  - apis
+github:
+  path: 02-Calling-an-API
+sample_download_required_data:
+  - client
+  - api
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD002 MD041 -->
+
+Most single-page apps use resources from data APIs. You may want to restrict access to those resources so that only authenticated users with sufficient privileges can access them. Auth0 lets you manage access to these resources using [API Authorization](/api-auth).
+
+This tutorial shows you how to create a simple API using [Express](https://expressjs.com) that validates incoming JSON Web Tokens. You will then see how to call this API using an access token granted by the Auth0 authorization server.
+
+<%= include('../_includes/_calling_api_create_api') %>
+
+<%= include('../_includes/_calling_api_create_backend.md') %>
+
+Finally, modify `package.json` to add two new scripts `dev` and `api` that can be used to start the frontend and the backend API together:
+
+```json
+{
+  "name": "03-calling-an-api",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "dev": "npm-run-all --parallel serve api",
+    "api": "node server.js"
+  },
+
+  // .. package dependencies and other JSON nodes
+}
+```
+
+You can now start the project using `npm run dev` in the terminal, and the frontend Vue.js application will start up alongside the backend API.
+
+### Set up a proxy to the backend API
+
+To call the API from the frontend application, the development server must be configured to proxy requests through to the backend API. To do this, add a `vue.config.js` file to the root of the project and populate it with the following code:
+
+```js
+// vue.config.js
+
+module.exports = {
+  devServer: {
+    proxy: {
+      "/api": {
+        target: "http://localhost:3001"
+      }
+    }
+  }
+};
+```
+
+::: note
+This assumes that your project was created using [Vue CLI 3](https://cli.vuejs.org/guide/). If your project was not created in the same way, the above should be included as part of your Webpack configuration.
+:::
+
+With this in place, the frontend application can make a request to `/api/external` and it will be correctly proxied through to the backend API at `http://localhost:3001/api/external`.
+
+### Restart the application
+
+If you are already running the application using `npm run serve`, then stop that process for now. Restart the app using the following command, which will run both the frontend app and the backend API together:
+
+```bash
+PORT=3000 npm run dev
+```
+
+## Add the Audience
+
+The API backend that you created above expects API calls to have a JSON Web Token attached so that it can authorize the call. To do this, you must instruct Auth0 to emit an access token in JWT format. This is done by supplying the API identifier when your users authenticate, telling Auth0 which API you want an access token for.
```

Comment:
The API backend you created above expects API calls to have a JSON Web Token for authorization.